daemon/graphdriver/copy/copy.go

```diff
@@ -116,7 +116,7 @@ type dirMtimeInfo struct {
 //
 // The copyOpaqueXattrs controls if "trusted.overlay.opaque" xattrs are copied.
 // Passing false disables copying "trusted.overlay.opaque" xattrs.
-func DirCopy(srcDir, dstDir string, copyMode Mode, copyOpaqueXattrs bool) error {
+func DirCopy(srcDir, dstDir string, copyMode Mode, copyOpaqueXattrs bool, allowXattrFailure bool) error {
```

Comment:
Can we make this a daemon option?

Reply:
Yeah, I'd be okay with a storage-opt or something like that, so long as it's opt-in and flagged in `docker info` so we can identify if any bug reports are self-inflicted

Reply:
I agree. I'll check how to do it and make that option opted in by default.